src/shared/hooks/usePlugins.ts

```diff
@@ -52,13 +56,14 @@ export default function usePlugins() {
             }, manifest),
           })
         )
-        .catch(error =>
+        .catch(error => {
           setManifest({
             error,
             loading: false,
             data: null,
-          })
-        );
+          });
```

Comment:
May be we should make aborting a standard practice across all components that does API calls in UseEffect.